sched_slack_bot/views/schedule_dialog.py

```diff
@@ -84,7 +85,9 @@ def get_edit_schedule_block(
     schedule: Optional[Schedule] = None, callback: ScheduleDialogCallback = ScheduleDialogCallback.CREATE_DIALOG
 ) -> View:
     modal_type = CREATE_MODAL_TYPE if schedule is None else EDIT_MODAL_TYPE
+    # make sure external id is unique globally: https://github.com/slackapi/node-slack-sdk/issues/1012#issuecomment-684818059
     external_id = schedule.id if schedule is not None else CREATE_NEW_SCHEDULE_VIEW_ID
+    external_id = f"{external_id}-{str(uuid.uuid4())}"
```

Comment:
I believe by doing this, you are breaking the edit feature. It will use the external_id to decide which schedule to edit in the database. See https://github.com/Germandrummer92/SchedSlackBot/blob/main/sched_slack_bot/controller.py#L185 + https://github.com/Germandrummer92/SchedSlackBot/blob/main/sched_slack_bot/model/schedule.py#L149

Reply:
this came to my attention after the change :) now it works as adding a prefix uuid to the objectId and create a unique externalId, then in the schedule.py obtain the objectId suffix once again